apps/api/src/app/events/usecases/process-subscriber/process-subscriber.usecase.ts

```diff
@@ -34,11 +37,27 @@ export class ProcessSubscriber {
 
     const notification = await this.createNotification(command, template._id, subscriber);
 
-    const steps = matchMessageWithFilters(
+    let steps = matchMessageWithFilters(
       template.steps.filter((step) => step.active === true),
       command.payload
     );
 
+    const digests = steps.filter((step) => step.template.type === ChannelTypeEnum.DIGEST_BACKOFF);
+
+    if (digests.length > 0) {
+      const earliest = moment().subtract(10, 'minutes').toDate();
+      const jobs = await this.jobRepository.find({
+        updatedAt: {
+          $gte: earliest,
+        },
+        status: JobStatusEnum.COMPLETED,
```

Comment:
Worth adding here the subscriberId & the environmentId for a simple batching by subscriber and environment in this case.